### PR TITLE
Adding S6 documentation

### LINKED ISSUE #316: [Enhancement]: Add S6 documentation
### Instrument

Other (any stage)

### What is your suggestion?

Stage 6 currently has no documentation on the ECF page or the Outputs page on readthedocs. I will work on adding those now

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by taylorbell57**
Resolves #316 and updates all the rest of the ECFs in minor ways.

### CODE REVIEW

**Comment by taylorbell57** [comment]
Will also now close #300 as I've deleted that old, unused file

File: demos/JWST/S6_template.ecf [hunk 1]
```diff
@@ -11,26 +13,25 @@ x_unit          um # Options include any measurement of light included in astrop
...
 isplots_S6      5 # Generate few (1), some (3), or many (5) figures (Options: 1 - 5)
-testing_S6      False
 hide_plots      False # If True, plots will automatically be closed rather than popping up
 
 # Project directory
 topdir          /home/User/
 
-# Model to plot as well (path relative to project dir of a csv with col 1 as wavelength in microns, col 2 in the same units as the plotted spectrum, any headers preceded by a #)
-model_spectrum  None # Set to None if no model should be plotted
+# Model to plot underneath the fitted data points
```
Comment (kevin218) [inline demos/JWST/S6_template.ecf:28]: Normally, I plot the model **on top of** the data, but I can see instances where we might want to do plot the data underneath.  At some point, we should consider providing an option to modify the model's `zorder`.

File: docs/source/index.rst [hunk 2]
```diff
@@ -9,13 +9,14 @@ Welcome to Eureka!'s documentation!
...
 
-The code is separated into five parts or "Stages":
+The code is separated into six parts or "Stages":
 
 - Stage 1: An optional step that calibrates Raw data (converts ramps to slopes). This step can be skipped if you'd rather use STScI's JWST pipeline's Stage 1 outputs.
-- Stage 2: An optional step which calibrates Stage 1 data (performs flatfielding, unit conversion, etc.). This step can be skipped if you'd rather use STScI's JWST pipeline's  Stage 2 outputs.
-- Stage 3: Starts with Stage 2 data and reduces the data (performs background subtraction, etc.) in order to convert 2D spectra into a time-series of 1D spectra
-- Stage 4: Bins the 1D Spectra and generates light curves
-- Stage 5: Fits the light curves (under development)
+- Stage 2: An optional step which calibrates Stage 1 data (performs flatfielding, unit conversion, etc.). This step can be skipped if you'd rather use STScI's JWST pipeline's Stage 2 outputs.
```
Comment (kevin218) [inline docs/source/index.rst:15]: An optional step **that** calibrates...

File: demos/JWST/S6_template.ecf (hunk 1 above)
Comment (taylorbell57) [reply demos/JWST/S6_template.ecf:28]: I was biased the same way, but the problem is that the models will tend to be much higher resolution than the data (at least in my case), and as a result the data gets hidden. I'll open a new issue to allow the data to be plotted on top of or below the model

Comment (taylorbell57) [reply demos/JWST/S6_template.ecf:28]: For example:

fig6101_transmission

Comment (kevin218) [reply demos/JWST/S6_template.ecf:28]: Agreed, it all depends on the resolution of the model.

**Review by kevin218 — APPROVED** [review]

**Pull request merged**